VideoLocker/src/main/java/org/edx/mobile/view/adapters/DiscussionCommentsAdapter.java

```diff
@@ -187,20 +192,19 @@ public void addAll(List<DiscussionComment> items) {
         notifyItemChanged(lastCommentIndex); // Last item's background is different, so must be refreshed as well
     }
 
-    private static class ViewHolder extends RecyclerView.ViewHolder {
+    private static class ResponseAndCommentViewHolder extends RecyclerView.ViewHolder {
```

Comment:
Maybe this should be renamed to `ResponseOrCommentViewHolder` instead, which seems to be a more accurate description.